### PR TITLE
Schema changes

### PR DESCRIPTION
**Pull request by ilyakatz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.5.0
+
+* Validate that schema file is preset
+* Validate that objects are present in schema
+* Remove update validation, better update validation coming
+* Rake task to load schema (experimentatal)
```
Comment (andela-denoma) [inline CHANGELOG.md]: I think you meant experimental here

File: lib/restforce_mock/sandbox.rb [hunk 2]
```diff
@@ -18,7 +18,6 @@ def self.get_object(name, id)
 
     def self.update_object(name, id, attrs)
       current = storage[name][id]
-      validate_all_present_fields!(current, attrs)
```
Comment (andela-denoma) [inline lib/restforce_mock/sandbox.rb:21]: why was this removed? Does it mean there's no need for validation for sandbox?

**Comment by kaliara** [comment]
Didn't see anything that stood out.  Looks like restforce_mock is starting to grow up!

:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (ilyakatz) [inline CHANGELOG.md]: updating

File: lib/restforce_mock/sandbox.rb (hunk 2 above)
Comment (ilyakatz) [inline lib/restforce_mock/sandbox.rb:21]: see readme file "Remove update validation, better update validation coming"

Comment (ilyakatz) [inline lib/restforce_mock/sandbox.rb:21]: current validation is not correct, so it will produce wrong results. so i decided to remove it until there is a better way
